enterprise/server/content_addressable_storage_server_proxy/content_addressable_storage_server_proxy_test.go

```diff
@@ -259,6 +260,47 @@ func TestFindMissingBlobs(t *testing.T) {
 	expectNoAtimeUpdate(t, clock, requestCount)
 }
 
+func TestFindMissingBlobs_SkipRemote(t *testing.T) {
+	ctx := testContext()
+	remoteConn, _, _ := runRemoteCASS(ctx, testenv.GetTestEnv(t), t)
+	proxyEnv := testenv.GetTestEnv(t)
+	clock := clockwork.NewFakeClock()
+	proxyEnv.SetClock(clock)
+	proxyEnv.SetContentAddressableStorageClient(repb.NewContentAddressableStorageClient(remoteConn))
+	flags.Set(t, "cache_proxy.remote_atime_update_interval", atimeUpdatePeriod)
+	require.NoError(t, atime_updater.Register(proxyEnv))
+	proxyConn := runCASProxy(ctx, remoteConn, proxyEnv, t)
+	proxy := repb.NewContentAddressableStorageClient(proxyConn)
+
+	digestA := digestProto(fooDigest, 3)
+	digestB := digestProto(foofDigest, 4)
+	digestC := digestProto(barDigest, 3)
+
+	// Write digests A and B to the remote cache
+	remote := proxyEnv.GetContentAddressableStorageClient()
+	_, err := remote.BatchUpdateBlobs(ctx, updateBlobsRequest(map[*repb.Digest]string{digestA: "foo", digestB: "foof"}))
+	require.NoError(t, err)
+
+	// Write digests B and C to the local cache
```

Comment:
nit: write to proxy cache first so that you only have to write A to the remote?

Reply:
I wanted the local and remote caches to have slightly different data, which is why I'm writing directly to the internal cache here.